services/web/src/modals/EditUser.js

```diff
@@ -102,18 +98,20 @@ export default class EditUser extends React.Component {
               />
               {!this.isUpdate() && (
                 <Form.Input
-                  required
                   label="Password"
                   value={user.password || ''}
                   onChange={(e, { value }) => this.setField('password', value)}
                 />
               )}
+              {!this.isUpdate() && !this.state.password && (
+                <Message content="Not setting a password, will trigger an invitation to the user" />
```

Comment:
So any edits to the user will send them a new invitation? That's kind of what this implies, no?

Reply:
A temp password is set on creation, so this could only happen on creation